Library/Homebrew/Gemfile.lock

```diff
@@ -210,7 +210,7 @@ DEPENDENCIES
   yard-sorbet
 
 RUBY VERSION
-   ruby 3.3.7p123
+   ruby 3.3.8p144
 
 BUNDLED WITH
-   2.5.20
+   2.6.7
```

Comment:
This should match what we have in utils/gems.rb: https://github.com/Homebrew/brew/blob/35897fe49cfde1c3f7076768145da01f5180cfbc/Library/Homebrew/utils/gems.rb#L13.

`brew vendor-gems --update=--bundler` should do the right thing I think.

Reply:
I tried that without success. This was generated by removing Gemfile.lock and running `bundle install`.

Reply:
`bundle install` using Portable Ruby bundle or homebrew-core bundle?

Reply:
rbenv shims might have messed me up here.

Reply:
> Would be good to know what "without success" means - is it erroring?

Sorry for the ambiguity. It exited 0 with no changes.

Reply:
Fixed the Bundler version and also updated utils/gems.rb.